src/Components/Scenes/ViewRequests/RequestsInbox/RequestsInbox.js

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import RequestTableTitle from '../RequestTableTitle';
+import RequestTable from '../RequestTable';
+import { transformRequests } from '../../../../State/helpers';
+
+// Approver email is hard-coded until authentication is implemented
+const approverEmail = 'jonathan.delarosa@pnmac.com';
+const requestsInboxTableColumns = [
+  { name: 'Request', property: 'databasename' }, // placeholder from name property
+  { name: 'Description', property: 'description' },
+  { name: 'Status', property: 'requeststatus' },
+  { name: 'Date Requested', property: 'createddate' }
+];
+
+const RequestsInbox = props => {
+  const { requests, approverRequestsFetch } = props;
+
+  // Fetch all inbound requests given for approver's email
+  useEffect(() => {
+    approverRequestsFetch(approverEmail);
+  }, [approverRequestsFetch]);
+
+  const reqs = transformRequests(requests);
+
+  const setFooterButtonText = selected =>
+    `${selected.length} request${selected.length !== 1 ? 's' : ''} selected`;
+
+  return (
+    <>
+      <RequestTableTitle title="Requests Inbox" />
+      <RequestTable
+        tableColumns={requestsInboxTableColumns}
+        requests={reqs}
+        handleRequestClick={(e, id) => console.log(`request ${id} clicked`)}
```

Comment:
I get that these are temporary, but it may fail pipelines n such if it's a linter warning (our settings are pretty strict). Maybe `alert()` or something else, idk

If this actually doesn't cause a warning, then ignore

Reply:
tried testing the `lint-staged` function that is called with husky - there aren't any warnings on my end

Reply:
no warning on  my end also.